### PR TITLE
Fix for issue 162

### PR DESCRIPTION
**Pull request by TheProcedural**
## Context
- Change the mime detection package and add a "good enough" fallback for extension detection

This fixes the bug on issue (162)[https://github.com/aldinokemal/go-whatsapp-web-multidevice/discussions/162]

### CODE REVIEW

File: docker/golang.Dockerfile [hunk 1]
```diff
@@ -19,5 +19,10 @@ RUN apk update && apk add --no-cache ffmpeg
...
-# Run the binary.
-ENTRYPOINT ["/app/whatsapp"]
\ No newline at end of file
+
+# Set default environment variables for port and webhook
+ENV PORT 3000
+ENV WEBHOOK "http://localhost:3000/handler"
+
+# Use shell form to ensure environment variables are evaluated
+CMD ["/bin/sh", "-c", "/app/whatsapp --port ${PORT} -w=${WEBHOOK}"]
```
Comment (aldinokemal) [inline docker/golang.Dockerfile:28]: no need

**Comment by aldinokemal** [comment]
How do you demonstrate that doing this will resolve the problem?
Provide a screenshot or a video as proof here.

File: docker/golang.Dockerfile (hunk 1 above)
Comment (Erik-Bjerke) [reply docker/golang.Dockerfile:28]: This is something I did for myself and forgot to remove it from the pull request. Just ignore it

**Comment by aldinokemal** [comment]
Your PR modifies an unneeded file.

**Pull request closed without merging**